src/utils/proxy.js

```diff
@@ -52,6 +52,29 @@ const isExternal = function (match) {
   return match.to && match.to.match(/^https?:\/\//)
 }
 
+const stripOrigin = function (url) {
+  // Get the URL after http://host:port
+  return url.toString().replace(url.origin, '')
```

Comment:
`url.origin` does not include `username:password`, so this would fail if those are specified. I am not sure this is a possible case though in this instance.
What about `${url.pathname}${url.search}${url.hash}` instead?